main.tex

```diff
@@ -191,9 +193,11 @@ \subsection{The Moran Process}\label{sec:the_moran_process}
 themselves. This is a disadvantage for the Moran process where the best
 strategies cooperate well with other players using the same strategy.
 
-This work uses pair wise Moran processes in a similar way to matches in the many
+This work uses pair-wise Moran processes in a similar way to matches in the many
 IPD tournaments published since Axelrod's original work~\cite{Axelrod1980a}.
-This is a more robust tournament from an evolutionary point of view.
+This work offers a population-based perspective which adds additional
```

Comment:
Minor: looks like two sentences in a row that start with "This work" and we need a period at the end of the second.